solr-base.cfg

```diff
@@ -0,0 +1,211 @@
+# Do not extend from this configuration directly.
+# Extend one of:
+# - solr.cfg
+# - plone-development-solr.cfg
+
+[buildout]
+parts +=
+    solr-download
+    solr-instance
+
+
+[solr-settings]
+# port =
+host = 127.0.0.1
+base = /solr
+solr-major-version = 4
+solr-dist-version = 4.10.4
+solr-dist-md5sum = 8ae107a760b3fc1ec7358a303886ca06
+solr-cores = solr-default-plone-core
+solr-default-core = solr-default-plone-core
+
+# Do not move this to buildout:zcml-additional-fragments in this file
+# because buildout is buggy when using += and nested extends.
+zcml =
+    <configure xmlns:solr="http://namespaces.plone.org/solr">
+        <solr:connection host="${solr-settings:host}"
+                         port="${solr-settings:port}"
+                         base="${solr-settings:base}"/>
+    </configure>
+
+
+
+[solr-download]
+recipe = hexagonit.recipe.download
+url = https://archive.apache.org/dist/lucene/solr/${solr-settings:solr-dist-version}/solr-${solr-settings:solr-dist-version}.tgz
+md5sum = ${solr-settings:solr-dist-md5sum}
+strip-top-level-dir = true
+
+
+[solr-instance]
+recipe = collective.recipe.solrinstance:mc
+solr-location = ${solr-download:location}
+host = ${solr-settings:host}
+port = ${solr-settings:port}
+basepath = ${solr-settings:base}
+solr-version = ${solr-settings:solr-major-version}
+
+java_opts =
+  -server
+  -Xms256M
+  -Xmx1024M
+
+cores = ${solr-settings:solr-cores}
+default-core-name = ${solr-settings:solr-default-core}
+
+
+[solr-default-plone-core]
+<= solr-core-template
+
+
+[solr-core-template]
+max-num-results = 1000
+section-name = SOLR
+unique-key = UID
+logdir = ${buildout:directory}/var/solr
+default-search-field = SearchableText
+unique-key = UID
+spellcheckField = SearchableText
+default-operator = AND
+updateLog = true
+requestParsers-enableRemoteStreaming = true
+solrconfig-shards =
+
+extra-field-types =
+    <fieldType name="text_de" class="solr.TextField" positionIncrementGap="100">
+      <analyzer type="index">
+          <charFilter class="solr.MappingCharFilterFactory" mapping="mapping-ISOLatin1Accent.txt"/>
```

Comment:
I have seen that we sometimes have checked this file into our projects, but I do not do that I have one in `parts/solr-instance/solr/solr-default-plone-core/conf/mapping-ISOLatin1Accent.txt `.
What is the background?